packages/cactus-plugin-ledger-connector-fabric/src/main/json/openapi.json

```diff
@@ -368,6 +404,40 @@
                     }
                 }
             }
+        },
+        "/api/v1/plugins/@hyperledger/cactus-plugin-ledger-connector-fabric/get-prometheus-exporter-metrics": {
+            "get": {
+                "x-hyperledger-cactus": {
+                    "http": {
+                        "verbLowerCase": "get",
+                        "path": "/api/v1/plugins/@hyperledger/cactus-plugin-ledger-connector-fabric/get-prometheus-exporter-metrics"
+                    }
+                },
+                "operationId": "getPrometheusExporterMetricsEndpointV1",
+                "summary": "Get the Prometheus Metrics",
+                "parameters": [],
+                "requestBody": {
```

Comment:
@jagpreetsinghsasan Do we still need this in the request? IIRC HTTP GET requests cannot have a body.

Reply:
Fixed